tensorflow/lite/kernels/mul_test.cc

```diff
@@ -395,19 +395,26 @@ void WithBroadcast() {
   float kQuantizedTolerance = GetTolerance(-3.0, 3.0);
   std::vector<std::vector<int>> test_shapes = {
       {6}, {2, 3}, {2, 1, 3}, {1, 3, 1, 2}};
+  // Test with a smaller than 1 and greater than 1 quantization multiplier
+  std::vector<std::pair<float, float>> test_input_range = {{-3.0, 3.0},
```

Comment:
Nit: can you please make this a constant?

Reply:
Thank you for the review. I fixed that and added the `const` to the other variables of the test to stay coherent if it's alright for you.